Description: Be more careful when checking the "wrong password" output, there might be a new prompt before.
Author: Martin Pitt <martin.pitt@ubuntu.com>
Bug-Ubuntu: https://launchpad.net/bugs/298217 

diff -Nur -x '*.orig' -x '*~' libgksu-2.0.13~pre1//libgksu/libgksu.c libgksu-2.0.13~pre1.new//libgksu/libgksu.c
--- libgksu-2.0.13~pre1//libgksu/libgksu.c	2010-09-16 01:06:08.831166000 +0200
+++ libgksu-2.0.13~pre1.new//libgksu/libgksu.c	2010-09-16 01:07:09.451166009 +0200
@@ -3020,7 +3020,7 @@
 	  fprintf (stderr, "%s", buffer);
 	}
 
-      if (g_str_has_prefix (buffer, "Sorry, try again."))
+      if (g_str_has_prefix (buffer, "Sorry, try again.") || g_str_has_prefix (buffer, "GNOME_SUDO_PASSSorry, try again."))
 	g_set_error (error, gksu_quark, GKSU_ERROR_WRONGPASS,
 		     _("Wrong password."));
       else
